### PR TITLE
Upgrade Types

### PR DESCRIPTION
**Pull request by chrisdrackett**
Based on the current API Figma returns

### CODE REVIEW

File: src/figmaTypes.ts [hunk 1]
```diff
@@ -66,8 +139,8 @@ export interface Canvas extends Global {
 export interface FrameBase extends Global {
   /** An array of nodes that are direct children of this node */
   readonly children: ReadonlyArray<Node>;
-  /** Background color of the node */
-  readonly backgroundColor: Color;
```
Comment (vutran) [inline src/figmaTypes.ts:70]: `backgroundColor` is still available in their response.

**Review by vutran — CHANGES_REQUESTED** [review]
Everything LGTM besides that one potential breaking change.

File: src/figmaTypes.ts (hunk 1 above)
Comment (chrisdrackett) [reply src/figmaTypes.ts:70]: Can you get it to come back in a request? I found no way to cause this to be sent back, and the docs mention this is deprecated:

```
Background color of the node. This is deprecated, as frames now support more than a solid color as a background. Please use the background field instead.
```

**Comment by chrisdrackett** [comment]
@jongold curious what your thought are on getting this merged and published? the package I'm working on that uses these types is waiting on it :D

**Comment by jem-computer** [comment]
Shoot, didn't realize I hadn't published! My bad, will do that by tomorrow
at the latest :)

**Comment by jem-computer** [comment]
@chrisdrackett can you try 1.7.0? 🙏

**Comment by chrisdrackett** [comment]
@jongold looks good, type errors gone :) thanks!

I'll keep working on this as I go, I'm finding the Figma API docs don't align 100% with what the json actually returns. I'm guessing there are other things I'm missing

**Pull request merged**